File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportJava/Networking/src/main/java/transport/networking/dto/TripDTO.java
package transport.networking.dto;

import java.io.Serializable;

public class TripDTO implements Serializable {
    private Long id;
    private String destination;
    private String date;
    private String hour;
    private int availableSeats;

    public TripDTO(Long id, String destination, String date, String hour, int availableSeats) {
        this.id = id;
        this.destination = destination;
        this.date = date;
        this.hour = hour;
        this.availableSeats = availableSeats;
    }

    public Long getId() { return id; }
    public String getDestination() { return destination; }
    public String getDate() { return date; }
    public String getHour() { return hour; }
    public int getAvailableSeats() { return availableSeats; }
}

File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportC#/Network/dto/TripDTO.cs
using System.Text.Json.Serialization;

namespace FirmaTransportC_.Networking.dto
{
    [Serializable]
    public class TripDTO
    {
        public long Id { get; set; }
        public string Destination { get; set; }
        public string Date { get; set; }
        public string Hour { get; set; }
        public int AvailableSeats { get; set; }
        
        public TripDTO(long id, string destination, string date, string hour, int availableSeats)
        {
            Id = id;
            Destination = destination;
            Date = date;
            Hour = hour;
            AvailableSeats = availableSeats;
        }

        public TripDTO()
        {
        }
    }
}
